jasmine-standalone-3.6.0 2/src/bowling.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+class Bowling {
+  constructor(){
+    this._rolls = [];
+  }
+
+  roll(pins){
+    this._rolls.push(pins)
+  }
+
+  reset(){
+    this._rolls = [];
+  }
+
+    get score(){
+    let score = 0;
+    let rollIndex = 0;
+
+    for(let i = 0; i < 10; i++) {
+
+      if (this.isStrike(rollIndex)){
+        score += this.strikeBonus(rollIndex);
+        rollIndex++;
+        continue;
+      }
+      const frameScore = this._rolls[rollIndex] + this._rolls[rollIndex + 1];
+
+      if (this.isSpare(frameScore)){
+        score += this.spareBonus(rollIndex);
+      } else {
+        score += frameScore;
+      }
+
+
+      rollIndex += 2;
+    }
+    return score;
+  }
+
+isSpare(frameScore){
```

Comment:
I think indentation for lines 41-55 might need adjusting (to the right)

Reply:
Thanks , I'll do it